### ISSUE TITLE
Suggestion: Data separate from the program

### ISSUE THREAD

**Issue by Urvdmih** [Just2good/TFT-Overlay#114]
I think you need separate data from program in the json or xml files. Like items-9.16.json, champs-9.16.json, because patches come out often, and it’s easier to update separately files online, without updating whole program. 
And data may be always up to date in this case.

**Comment by tim-sheung** [comment]
> I think you need separate data from program in the json or xml files. Like items-9.16.json, champs-9.16.json, because patches come out often, and it’s easier to update separately files online, without updating whole program.
> And data may be always up to date in this case.

Agree. Perhaps give the flexibility for user to manually import these files

**Comment by xadamxk** [comment]
@tim-gnue At that point there would be no difference than just downloading the new program.
Once an API comes out for TFT, if an API releases, all the developer has to do is integrate a polling system to get official data from.
